test/views/declaration/spec/UnitViewSpec.scala

```diff
@@ -66,3 +67,26 @@ object MessagesKeyMatcher {
 object UnitViewSpec extends Injector {
   val realMessagesApi: MessagesApi = instanceOf[MessagesApi]
 }
+
+/*
+    Fails the test if a view is configured with a message key that doesnt exist in the messages file
+ */
+private class AllMessageKeysAreMandatoryMessages(msg: Messages) extends Messages {
```

Comment:
Instead of comment I'll change the name to be better

Reply:
Any suggestions.  Copied this from Internal UI.  I think the name is OK.  I'll get rid of the comment

Reply:
I think it'll be better `AllMessageKeysAreMandatory`, but in practice we're not extending this very often so this name is not a problem.